src/Umbraco.Cms.Integrations.Commerce.Shopify/Services/IShopifyService.cs

```diff
@@ -13,6 +13,10 @@ public interface IShopifyService
 
         void RevokeAccessToken();
 
-        Task<ResponseDto<ProductsListDto>> GetResults();
+        Task<ResponseDto<ProductsListDto>> GetResults(string pageInfo);
```

Comment:
This will be a breaking change, which is fine for this component, but bump the version to a new minor.